src/AreaChart/AreaChart.jsx

```diff
@@ -1,81 +1,22 @@
-import React, {Component, PropTypes} from 'react'
-import ReactHighcharts from 'react-highcharts'
+import React, {Component} from 'react'
+import createChart, {createDefaultChart} from '../Chart'
+import {Line} from 'react-chartjs'
 
-const baseConfig = {
-  chart: {
-    type: 'area'
-  },
-  title: {
-    text: 'US and USSR nuclear stockpiles'
-  },
-  subtitle: {
-    text: 'Source: <a href="http://thebulletin.metapress.com/content/c4120650912x74k7/fulltext.pdf">' +
-    'thebulletin.metapress.com</a>'
-  },
-  xAxis: {
-    allowDecimals: false,
-    labels: {
-      formatter: function () {
-        return this.value
-      }
-    }
-  },
-  yAxis: {
-    title: {
-      text: 'Nuclear weapon states'
-    },
-    labels: {
-      formatter: function () {
-        return this.value / 1000 + 'k'
-      }
-    }
-  },
-  tooltip: {
-    pointFormat: '{series.name} produced <b>{point.y:,.0f}</b><br/>warheads in {point.x}'
-  },
-  plotOptions: {
-    area: {
-      pointStart: 1940,
-      marker: {
-        enabled: false,
-        symbol: 'circle',
-        radius: 2,
-        states: {
-          hover: {
-            enabled: true
-          }
-        }
-      }
-    }
-  }
-}
-
-export class AreaChart extends Component {
+class AreaChart extends Component {
   static propTypes = {
-    drilldown: PropTypes.object,
-    series: PropTypes.array.isRequired,
-    title: PropTypes.string.isRequired
-  }
-
-  static defaultProps = {
-    drilldown: {}
+    ...createChart.propTypes
   }
-
+  
   render () {
-    const {drilldown, series, title} = this.props
-    const config = {
-      ...baseConfig,
-      drilldown,
-      series,
-      title: {text: title}
-    }
-
     return (
-      <ReactHighcharts
-        config={config}
-        drilldown={drilldown}
-        series={series}
+      <Line
+        {...this.props}
       />
     )
   }
-}
\ No newline at end of file
+}
+
+const wrappedChart = createChart(AreaChart)
+
+export default wrappedChart
```

Comment:
I would consider changing `createChart`s name to `Chart` and `createDefaultChart` to `DefaultChart` and then making it a decorator. You can look at how [`Hydrateable`](https://github.com/jeffshaver/safe-app/blob/dev/src/js/decorators/Hydrateable.jsx) is implemented in safe-app.

then you could `export default class AreaChart {`

and then at the bottom you can export the default one.